api/user/userRouter.js

```diff
@@ -67,7 +67,7 @@ router
 
         dbModel.editUser(id, body) 
             .then(user => {
-                res.status(200).json(body)
+                res.status(200).json(user)
```

Comment:
Does this work? I've never had it work when it wasn't body.

Reply:
Yep! It's needed for the payload through Redux